src/contracts/pods/EigenPod.sol

```diff
@@ -584,15 +706,21 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         returns (VerifiedWithdrawal memory)
     {
         uint64 withdrawalTimestamp = withdrawalProof.getWithdrawalTimestamp();
-        bytes32 validatorPubkeyHash = validatorFields.getPubkeyHash();
 
         /**
-         * Withdrawal processing should only be performed for "ACTIVE" or "WITHDRAWN" validators.
-         * (WITHDRAWN is allowed because technically you can deposit to a validator even after it exits)
+         * Withdrawal processing should only be performed for "ACTIVE" validators
+         * WITHDRAWN validators will have their ETH automatically sent to this contract
+         * and can exit these funds via the partial withdrawal flow.
          */
         require(
-            _validatorPubkeyHashToInfo[validatorPubkeyHash].status != VALIDATOR_STATUS.INACTIVE,
-            "EigenPod._verifyAndProcessWithdrawal: Validator never proven to have withdrawal credentials pointed to this contract"
+            _validatorPubkeyHashToInfo[validatorPubkeyHash].status == VALIDATOR_STATUS.ACTIVE,
```

Comment:
oh i think this might get rid of my edge case above. the issue now is that its unclear what happens to pods which have validators that have never been repointed but have fully withdrawn. 

i think this is a common enough case that we can't say "don't do that" like we would for redeposits...

Reply:
I'm gonna take "never been repointed" to mean that they haven't verified withdrawal credentials?

The answer is that that ether is "unaccounted for" and will be included in the `eligibleBalance` calculated when you start a partial withdrawal.

So they can be withdrawn through the checkpoint process.

Reply:
ah yep, sounds good